cpp/daal/src/externals/service_math_mkl.h

```diff
@@ -135,27 +101,57 @@ struct MklMath<double, cpu>
         return r;
     }
 
-    static void vPowx(SizeType n, const double * in, double in1, double * out) { VMLFN_CALL(dPowx, HAynn, ((int)n, in, in1, out)); }
+    static void vPowx(SizeType n, const double * in, double in1, double * out)
+    {
+        __DAAL_MKLFN_CALL_MATH(vmdPowx, ((int)n, in, in1, out, (VML_LA | VML_FTZDAZ_ON | VML_ERRMODE_STDERR)));
```

Comment:
The precision was dropped a bit for all functions with this change. It was VML_HA (i.e. "high accuracy"), and now it is VML_LA ("low accuracy").
Maybe not a big deal for machine learning algorithms. But maybe it contributed to some test failures in sklex.
Need to return to this after RC.
```suggestion
        __DAAL_MKLFN_CALL_MATH(vmdPowx, ((int)n, in, in1, out, (VML_HA | VML_FTZDAZ_ON | VML_ERRMODE_STDERR)));
```